g/go-control-plane/go-control-plane_v0.9.0_RHEL_8.4.sh

```diff
@@ -0,0 +1,130 @@
+# -----------------------------------------------------------------------------
+#
+# Package       : github.com/envoyproxy/go-control-plane
+# Version       : v0.9.0
+# Source repo   : https://github.com/go-stack/stack.git
+# Tested on     : RHEL 8.4
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Amit Baheti (aramswar@in.ibm.com)
+#
+# Disclaimer    : This script has been tested in root mode on given
+# ==========    platform using the mentioned version of the package.
+#               It may not work as expected with newer versions of the
+#               package and/or distribution. In such case, please
+#               contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=github.com/envoyproxy/go-control-plane
+PACKAGE_VERSION=v0.9.0
+PACKAGE_PATH=https://github.com/envoyproxy/go-control-plane.git
+
+yum -y update && yum install -y nodejs nodejs-devel nodejs-packaging npm python38 python38-devel ncurses git jq wget gcc-c++
+
+# Install Go and setup working directory
+wget https://golang.org/dl/go1.16.12.linux-ppc64le.tar.gz && \
+    tar -C /bin -xf go1.16.12.linux-ppc64le.tar.gz && \
+    mkdir -p /root/go/src
+
+export PATH=$PATH:/bin/go/bin
+export GOPATH=/root/go
+
+mkdir -p output
+
+OS_NAME=$(cat /etc/os-release | grep ^PRETTY_NAME | cut -d= -f2)
+
+export PATH=$GOPATH/bin:$PATH
+export GO111MODULE=on
+
+function test_with_master_without_flag_u(){
```

Comment:
it's okay to simplify the template to use only specific set of instructions needed for building/test the particular package